src/HtmlConverter.php

```diff
@@ -133,6 +133,8 @@ private function convertChildren(ElementInterface $element)
             return;
         }
 
+        $this->preProcess($element);
```

Comment:
Do you think it would be beneficial to use an event dispatcher in these instances?